launch/launch/logging/__init__.py

```diff
@@ -300,33 +315,32 @@ def set_log_format(self, log_format, *, log_style=None):
         else:
             self.file_formatter = None
 
-    def get_log_file_path(self, file_name='launch.log'):
+    def get_log_file_path(self, log_file_name='launch.log'):
```

Comment:
this still breaks the user application, we should keep `file_name` here.

```suggestion
    def get_log_file_path(self, file_name='launch.log'):
```